src/main/java/com/matera/cadastrocentral/address/AddressService.java

```diff
@@ -10,6 +10,11 @@
 @Service
 public class AddressService {
 
+    @Autowired
+    public AddressService(AddressRepository addressRepository) {
+        this.addressRepository = addressRepository;
+    }
+
     @Autowired
```

Comment:
Since you are injeting on the constructor, can you please remove this `@Autowired` here, since it's duplicated